.nvmrc

```diff
@@ -0,0 +1 @@
+8.9.1
```

Comment:
Is this necessary to be committed to the repo?

Reply:
Not required, but helpful for most macOS / *nix users who use nvm (it will auto-select the node version).

Reply:
Recommend Dev version, not version required for using